plugins/SoundFontPlayer/SoundFontPlayer.cpp

```diff
@@ -244,7 +244,7 @@ Sf2Instrument::~Sf2Instrument()
 
 
 
-void Sf2Instrument::saveSettings( QDomDocument & _doc, QDomElement & _this )
+void SoundFontInstrument::saveSettings( QDomDocument & _doc, QDomElement & _this )
```

Comment:
(Non-blocking) Per the Coding Conventions [re: parentheses](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces), [re: pointers](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers), [re: function parameters](https://github.com/LMMS/lmms/wiki/Coding-Conventions#function-parameters):

> 1. Spaces MUST NOT be added after an opening parenthesis or bracket
> 2. Spaces MUST NOT be added before a closing parenthesis or bracket

and,

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

and, 

> Some older code prefixed function parameters with an underscore `_`. Function parameters SHOULD NOT be prefixed with an underscore.

```suggestion
void SoundFontInstrument::saveSettings(QDomDocument& doc, QDomElement& el)
```

This will require updating `_this` to `el` in the function body.